analyze_logs.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -ex # -e: exit early on any error. -x: write all commands to stderr
 
+date '+%Y-%m-%d'
```

Comment:
This is the line the monitor will look for in the analysis logs.

Reply:
...in hindsight, I should make this 1-line change a separate PR that can be merged in quickly, so the canaries are ready to be monitored when this PR is approved.